style.css

```diff
@@ -0,0 +1,104 @@
+body,
+h1,
+h2,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  padding-left: 24px;
+  padding-right: 24px;
+  display: flex;
+}
+
+nav {
+  width: 98%;
+  height: 60px;
+  background: #3c3a39;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.navbar-logo {
+  display: flex;
+  font-size: 18px;
+  line-height: 24px;
+  font-weight: 700;
+  font-family: Roboto, sans-serif;
+  color: #fff;
+  text-decoration: none;
+}
+
+.navbar-menu {
+  padding-right: 50px;
+}
+
+.header-box {
+  background-size: 256px;
+  height: 600px;
+  width: 100%;
+  background-color: #1c1a19;
+  display: flex;
+  background-repeat: no-repeat;
+  background-position: bottom right -110px;
+  background-image: url("./Images/header-illustration.png");
+}
+

```

Comment:
- [ ] Kindly remove the `height: 600px;` role, to remove the white space at the bottom, as required by the previous reviewer.

![image](https://user-images.githubusercontent.com/35535130/158914448-17813183-9cc9-467a-9987-c9bc806cc3cf.png)